MCLOUD-7120: Add command which validate .magento.env.yaml locally before push (#8486)
magento/devdocs: src/cloud/project/magento-env-yaml.md

  deploy:
    CLEAN_STATIC_FILES: false
```

## Validate configuration file

> To validate the configuration file before pushing it to the environment you can use the command:

Reviewer: ```suggestion
Use the following CLI command to validate the `.magento.env.yaml` configuration file before pushing changes to the remote Cloud environment.
```

> To validate the configuration file before pushing it to the environment you can use the next command:

Reviewer 2: ```suggestion
To validate the configuration file before pushing it to the environment you can use the command:
```

```bash
php ./vendor/bin/ece-tools cloud:config:validate
```

> The command will show a list of problems in the `.magento.env.yaml` file if the configuration is not valid. 

Reviewer: It would be better to provide a sample response that contains errors or problems and advice on fixing the errors.

Author: The response example was added

Reviewer: That is perfect! Thanks!

  deploy:
    CLEAN_STATIC_FILES: false
```

## Validate configuration file

Use the following CLI command to validate the `.magento.env.yaml` configuration file before pushing changes to the remote Cloud environment.

```bash
php ./vendor/bin/ece-tools cloud:config:validate
```

> The command will show a list of problems in the `.magento.env.yaml` file if the configuration is not valid, for example:

Reviewer: ```suggestion
The following sample response provides a list of items to correct:
```
